### PR TITLE
chore: project.id, configFile in reporter apis

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/test-api/class-testconfig.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ const config: PlaywrightTestConfig = {
 export default config;
 ```
 
+## property: TestConfig.configFile
+* since: v1.27
+- type: ?<[string]>
+
+Path to config file, if any.
```
Comment (mxschmitt) [inline docs/src/test-api/class-testconfig.md:91]:
```suggestion
Absolute path to config file, if any.
```

File: utils/upload_flakiness_dashboard.sh [hunk 2]
```diff
@@ -95,7 +95,7 @@ az storage blob upload --connection-string "${FLAKINESS_CONNECTION_STRING}" -c u
 
 UTC_DATE=$(cat <<EOF | node
   const date = new Date();
-  console.log([date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate()].join('-'));
```
Comment (mxschmitt) [inline utils/upload_flakiness_dashboard.sh:98]: Looks unrelated.

**Review by dgozman — APPROVED** [review]
